docfx_yaml/extension.py

```diff
@@ -758,12 +758,107 @@ def _extract_docstring_info(summary_info, summary, name):
             words = []
         else:
             words.append(word)
-    
+
         index += 1
 
     return top_summary
 
 
+def reformat_summary(summary: str) -> str:
+    """Applies any style changes to be made specifically for DocFX YAML.
+
+    Makes the following changes:
+      - converts ``text`` to `text`
+      - [add any additional changes to be made]
+
+    Args:
+        summary: The summary to be modified.
+
+    Returns:
+        Converted summary suitable for DocFX YAML.
+    """
+
+    reformatted_lines = []
+
+    single_backtick = '`'
+    double_backtick = '``'
+    triple_backtick = '```'
+    for line in summary.split('\n'):
+        # Check that we're only looking for double backtick (``) and not
+        # comments (```).
+        if triple_backtick not in line and double_backtick in line:
+            reformatted_lines.append(line.replace(double_backtick, single_backtick))
+
+        # Add any additional changes to be made here.
+        #elif:
+
+        else:
+            reformatted_lines.append(line)
+
+    return '\n'.join(reformatted_lines)
+
+
+def _reformat_codeblocks(content: str) -> str:
+    """Formats codeblocks from ``` to <pre>."""
+    codeblock_fence = '```'
+    codeblock_current = '<pre>'
+    codeblock_next = '</pre>'
```

Comment:
Might just be me, but I find the `codeblock` prefix kind of confusing in this. We're in the `codeblock` function, why do we need this prefix? And I'd expect a different prefix for the fence than the HTML tags.

Especially when the there is a `triple_backtick` variable above. That name is clear to me.

Reply:
Agreed - I shouldn't have to go back and forth between the names. Done.